### DISCUSSION TITLE
Station list for GA fails.

### DISCUSSION THREAD

**Discussion by kgrunenberg** [weather-gov/api#285] [Q&A]
I am trying to build a cache of stations by state but requests for the GA station list keep failing. I suspect this is related to the issue #91 and there are just too many stations in GA to parse.  How can I get a list of all stations in GA.

Thanks

Response from api:
{
    "correlationId": "9e369e7a",
    "title": "Unexpected Problem",
    "type": "https://api.weather.gov/problems/UnexpectedProblem",
    "status": 500,
    "detail": "An unexpected problem has occurred.",
    "instance": "https://api.weather.gov/requests/9e369e7a"
}

**Comment by KTibow** [comment]
Just get all the state codes, and do it for each state as in #91.

**Comment by kgrunenberg** [comment]
@KTibow That is whats failing. When I request the stations for GA I get the 500 error

**Comment by scadergit** [comment]
Turns out there is a station in GA that has bad geometry and is throwing an error on the entire response. It should simpy ignore bad geometry. I created a support issue to resolve, but we have a laundry list we're working through.

**Comment by kgrunenberg** [comment]
@scadergit Thank you. Can tell me the id of the offending station so I can work around it for now.

**Comment by scadergit** [comment]
I looked but the logs don't provide which station it stopped on. Maybe @StephenClouse has it running locally to determine the station.

**Comment by kgrunenberg** [comment]
Thanks I have been able to work around it for now.

**Comment by jah1007** [comment]
The offending stations have been removed and this GA station list should now be available.
